cs/unittest/RunTests.tt

```diff
@@ -16,7 +16,7 @@ var lines = File.ReadAllLines(Path.Combine(testRoot, "RunTests"))
     .ToList();
 
 var skipList = new[] { 13, 32, 39, 258, 40, 259, 41, 260, 59, 60, 61, 66, 68, 90,
-	25, 26, 349, 350, 356, // crash
+	25, 26, 349, 350, 356, 357, 358, // crash
```

Comment:
this has interesting implications for consumers, since before we would not crash, and default to X